### ISSUE TITLE
HPE StoreOnce Catalyst integration feature_request

### ISSUE THREAD

**Issue by pogo1975** [olahallengren/sql-server-maintenance-solution#53]
Hi

Is it possible for you to add integration similar to DataDomain for this product:

HPE StoreOnce Catalyst
https://support.hpe.com/hpsc/doc/public/display?docId=emr_na-c04633655

Regards
Jarek

**Comment by olahallengren** [comment]
The way the integration with Data Domain Boost and the other backup products works, is that I am executing extended or CLR stored procedures from the vendor. Do you know if that is possible with HPE StoreOnce Catalyst?

**Comment by pogo1975** [comment]
Yes i think so it will work the same way, in attached pdf from page 60 are
supported commands with extended procedures, if you want to try, i or you
can yourself install Storeonceappliance trial.

Regards
Jarek

2018-06-04 21:34 GMT+02:00 Ola Hallengren <notifications@github.com>:

**Comment by olahallengren** [comment]
Thank you. This looks very good.

**Comment by olahallengren** [comment]
The challenge with adding support for more backup products is that testing is getting more complex, and that it is more work to add new features.

Please vote (+1) for this issue if you are using HPE StoreOnce Catalyst to back up your SQL Servers, and would be interested in integration with my scripts.

**Comment by dataartisan** [comment]
Curious if is feature built/supported?
